app/components/live-sidebar.js

```diff
@@ -12,6 +12,10 @@ export default class LiveHeaderComponent extends Component {
     this.isSideBarOpen = !this.isSideBarOpen;
   }
 
+  @action sidebarPanel() {
```

Comment:
This might not be needed here as you've passed the prop you  can directly use with onclick like,
`{{on 'click' @sidebarPanel}}`

Reply:
fixed